tests/test_crossbar.py

```diff
@@ -110,14 +110,14 @@ def test_place_aquire(place):
 
 def test_place_add_no_name(crossbar):
     with pexpect.spawn('python -m labgrid.remote.client create') as spawn:
-        spawn.expect("missing place name")
+        spawn.expect("missing place name. Set with -p <place> or via env var $PLACE")
```

Comment:
This fails on travis. Maybe because you need to escape some parts which are interpreted as regular expressions?

Reply:
Yes, that's my assumption, too when I looking at the test results. Will fix it an re-push it.

Reply:
The test expects an EOF after missing place name, thats why it fails.

Reply:
expect(pattern) discards any unmatched output before pattern matches, so I don't think expect(pexpect.EOF) is the problem. Also, according to the backtrace, the error is before the EOF line.

Reply:
I read it as 'somepatter''EOF', my bad.